pgmpy/tests/test_models/test_LinearGaussianBayesianNetwork.py

```diff
@@ -195,5 +195,22 @@ def test_get_random_cpds(self):
         cpds = model_lin.get_random_cpds()
         self.assertEqual(len(cpds), len(model.nodes()))
 
+    def test_get_random(self):
+        model = LinearGaussianBayesianNetwork()
+        random_model = model.get_random(
```

Comment:
Because `get_random` is a static method, you should be able to call it directly using `LinearGaussianBayesianNetwork.get_random` instead of first initializing the object. I would suggest to directly use `LinearGaussianBayesianNetwork.get_random` instead of `model.get_random`

Reply:
Should i add more models with different values of parameters and check assertion for all of them?

Reply:
Yes, I would just check whether the arguments work as expected, for example, check if we provide `node_names`, the final model has the same node names. Some of the arguments can be more difficult to test for, for example, loc or sd as they are then used for generating random numbers, and hence would return a different value every time. For those you can specify the seed argument, so that every time the tests are run you get the same value from the random number generator.

Reply:
Will there be any floating point difference in CPDs when generated randomly. Should I use np.allclose instead of np.array_equal. And if yes, what could be the tolerance.